extensions/blocks/instagram-gallery/edit.js

```diff
@@ -218,6 +211,12 @@ const InstagramGalleryEdit = props => {
 						</PanelRow>
 					</PanelBody>
 					<PanelBody title={ __( 'Gallery Settings', 'jetpack' ) }>
+						<div className="wp-block-jetpack-instagram-gallery__count-notice">{ noticeUI }</div>
```

Comment:
This LGTM, but I'd probably remove the horizontal margins (and maybe increase the bottom one?) 🤔 

(I'll point out where to adjust it in another comment)

| Before | After |
| - | - |
| <img width="277" alt="Screenshot 2020-04-09 at 11 39 35" src="https://user-images.githubusercontent.com/2070010/78887090-30576a00-7a57-11ea-8ed2-ec1647f730a6.png"> | <img width="275" alt="Screenshot 2020-04-09 at 11 39 25" src="https://user-images.githubusercontent.com/2070010/78887102-34838780-7a57-11ea-8bf5-265e6cc998ad.png"> |

Reply:
much nice, done - also made copied @scruffian's [approach and made it non-dismissable ](https://github.com/Automattic/jetpack/pull/15424/files#diff-60345e7f8e6630a59bdb18d2fde12b3bR87)which makes sense I think with 'info' notices.